packages/edit-site/src/components/global-styles/font-library-modal/installed-fonts.js

```diff
@@ -114,164 +113,179 @@ function InstalledFonts() {
 	return (
 		<div className="font-library-modal__tabpanel-layout">
 			{ isResolvingLibrary && (
-				<HStack align="center">
-					<Spacer />
-					<Spinner />
-					<Spacer />
-				</HStack>
+				<div className="font-library-modal__loading">
+					<HStack
+						justify="center"
+						style={ {
+							height: '100%',
+						} }
+					>
+						<ProgressBar />
+					</HStack>
+				</div>
```

Comment:
```suggestion
				<div className="font-library-modal__loading">
					<ProgressBar />
				</div>
```

I think this component could be made much easier.

In addition to this, is it possible to achieve this by adding the following style?

```
.font-library-modal__loading {
	align-items: center;
	justify-content: center;
}
```

These styles are similar to [the site editor's loader](https://github.com/WordPress/gutenberg/blob/890fa18f8738b529e5896f00ebfbc1d1d08bc560/packages/edit-site/src/components/canvas-loader/style.scss#L9-L10).

Reply:
Ah nice, thank you for pointing out the site editor's loader CSS. I think this works nicely, I made these changes here: https://github.com/WordPress/gutenberg/pull/60570/commits/ab509646306041077346ce9efdcc7ee42f6e88a6.